ajax/libs/grapesjs/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "grapesjs",
+  "filename": "grapes.min.js",
+  "description": "Free and Open Source Web Builder Framework",
+  "version": "0.9.13",
+  "author": "Artur Arseniev",
+  "license": "BSD-3-Clause",
+  "homepage": "https://grapesjs.com",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/artf/grapesjs.git"
+  },
+  "keywords": [
+    "wte",
+    "grapes",
+    "grapesjs",
+    "web template editor",
+    "web site builder",
+    "web template builder",
+    "html website builder",
+    "site builder",
+    "newsletter builder",
+    "wysiwyg",
+    "template",
+    "editor"
+  ],
+  "npmName": "grapesjs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/!(fontawesome*|FontAwesome*)"
```

Comment:
How about this `**/!(fontawesome|FontAwesome)*`?
What do you think?
Thank you.